### PR TITLE
Improving help text

### PR DESCRIPTION
**Pull request by goravjoshi**

### CODE REVIEW

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/help-secret.html [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <div>
-    <p>This is the value of the token that was displayed when the <a
-            href="https://confluence.atlassian.com/display/BitbucketServer/Personal+access+tokens">Bitbucket personal
-        access token</a> was created in Bitbucket Server. It allows users to automatically set up build triggers when
-        creating a Jenkins project. Importantly, for this to work the token must have project admin permissions.</p>
-    <p>Users will only be able to use the token to set up build triggers.</p>
+    <p><a href="https://confluence.atlassian.com/display/BitbucketServer/Personal+access+tokens">Personal access
+        token</a> are generated in Bitbucket Server. The token you add must also have project admin permissions
```
Comment (dkjellin) [inline src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/help-secret.html]:
```suggestion
        tokens</a> are generated in Bitbucket Server. The token you add must have project admin permissions
```

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/help-secret.html [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
+        for the projects that you want Jenkins to build from.</p>
+
+    <p>Providing this token will allow your users to automatically set up build triggers when creating Jenkins jobs.
+        They won't be able to use it for anything else.</p>
```
Comment (dkjellin) [inline src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/help-secret.html]: This is strictly speaking not true, it is also used for build-status posting. If we're happy to lie a little bit here I'm happy to leave it.

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-credentialsId.html [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
-        do this, it needs credentials with access to the projects and repositories you want it to build from.</p>
+    <p>When running a job, Jenkins will use these credentials to check out the source code for builds. For Jenkins to do
+        this, it needs credentials with access to the projects and repositories you want it to build from.</p>
 
     <p>To provide Jenkins with credentials you can:</p>
     <ul>
-        <li>Choose credentials from the list.</li>
-        <li>Add credentials as a username and password.</li>
+        <li>choose credentials from the list</li>
+        <li>add credentials as a username and password (for the password, you can enter a Bitbucket Server password or a
```
Comment (dkjellin) [inline src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-credentialsId.html]:
```suggestion
        <li>add credentials as a <strong>Username with password</strong> (for the password, you can enter a Bitbucket Server password or a
```
? so we are consistent with our other documentation?

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/help-secret.html (hunk 2 above)
Comment (goravjoshi) [reply src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/help-secret.html]: Yup, @ctan-atlassian and I have discussed about this. We decided to keep it as it is now.

**Pull request merged**